aptos-move/vm-genesis/src/lib.rs

```diff
@@ -82,6 +84,7 @@ pub fn encode_genesis_transaction(
     framework: &ReleaseBundle,
     chain_id: ChainId,
     genesis_config: GenesisConfiguration,
+    initial_balances: &[InitialBalance],
```

Comment:
I wonder if we can add this to GenesisConfiguration? We'll have a lot more things later on like delegations, vesting, etc. so we would end up with quite a long list of args here.